docker/lr-dv/Makefile

```diff
@@ -0,0 +1,12 @@
+VERSION = 1.3.0
+TAG1 = us.gcr.io/broad-dsp-lrma/lr-deepvariant:$(VERSION)
```

Comment:
Nice. We should make a plan to retire all of our quay.io uploads now. They're just doubling our Docker build times for no good reason.

Reply:
we need human-power... :wink: